src/Params.ml

```diff
@@ -235,7 +261,7 @@ type if_then_else =
 
 let get_if_then_else (c : Conf.t) ~first ~last ~parens ~parens_bch
     ~parens_prev_bch ~xcond ~expr_loc ~fmt_extension_suffix ~fmt_attributes
-    ~fmt_cond ~exp_grouping =
+    ~fmt_cond ~exp_grouping ~exp_grouping_bch =
```

Comment:
Adding again a new argument to this monster function. Formatting of if-then-else is too complex and not defined.